apps/web/features/editor/blocks/custom-code-block.tsx

```
@@ -187,6 +187,7 @@ export const customCodeBlockSpec: any = createReactBlockSpec(
 											overflowWrap: 'break-word',
 										}}
 									>
+										{/* PrismJS escapes HTML entities, preventing XSS */}
 										<code dangerouslySetInnerHTML={{ __html: highlight }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)